ReactSkia/components/RSkComponent.h

```diff
@@ -6,6 +6,7 @@
 #include "include/core/SkCanvas.h"
 #include "react/renderer/mounting/ShadowView.h"
 #include "react/renderer/components/view/ViewProps.h"
+#include "react/renderer/components/image/ImageShadowNode.h"
```

Comment:
ImageShadowNode.h include required ?

Reply:
ImageShadowNode.h header file required for ImageresizeMode but not in RSkComponent.h file it required in RSkComponentImage.h file.